src/hotspot/share/opto/predicates.cpp

```diff
@@ -798,32 +803,37 @@ InitializedAssertionPredicateCreator::InitializedAssertionPredicateCreator(Phase
 //                         success     fail path                     new success   new Halt
 //                           proj    (Halt or UCT)                       proj
 //
-IfTrueNode* InitializedAssertionPredicateCreator::create_from_template(IfNode* template_assertion_predicate,
-                                                                       Node* new_control, Node* new_init,
-                                                                       Node* new_stride) {
+InitializedAssertionPredicate InitializedAssertionPredicateCreator::create_from_template(
+    IfNode* template_assertion_predicate, Node* new_control, Node* new_init, Node* new_stride) const {
   OpaqueInitializedAssertionPredicateNode* assertion_expression =
       create_assertion_expression_from_template(template_assertion_predicate, new_control, new_init, new_stride);
-  return create_control_nodes(new_control, template_assertion_predicate->Opcode(), assertion_expression,
-                              template_assertion_predicate->assertion_predicate_type());
+   IfTrueNode* success_proj = create_control_nodes(new_control, template_assertion_predicate->Opcode(),
+                                                   assertion_expression,
+                                                   template_assertion_predicate->assertion_predicate_type());
+  return InitializedAssertionPredicate(success_proj);
```

Comment:
```suggestion
  IfTrueNode* success_proj = create_control_nodes(new_control,
                                                  template_assertion_predicate->Opcode(),
                                                  assertion_expression,
                                                  template_assertion_predicate->assertion_predicate_type());
  return InitializedAssertionPredicate(success_proj);
```

Reply:
Indentation, and: If you split the args over lines, I would at least split all of them consistently.

Reply:
I usually just make sure that long lines split but don't enforce a single arg per line. But I don't mind adapting to that here :-)